tests/test_downloader_handlers.py

```diff
@@ -687,9 +687,6 @@ class BaseFTPTestCase(unittest.TestCase):
     password = "passwd"
     req_meta = {"ftp_user": username, "ftp_password": password}
 
-    if six.PY3:
-        skip = "Twisted missing ftp support for PY3"
```

Comment:
The current tox/Travis use the latest Twisted version when Python 3 is used.
I don't think we need to skip the FTP tests, but we can do it if you think it's cleaner.